sklearn/linear_model/tests/ransac_iterations.py

```diff
@@ -0,0 +1,20 @@
+from sklearn import linear_model
```

Comment:
The test should be in `sklearn/linear_model/tests/test_ransac.py` and should be structured as a function with an assertion.